### DISCUSSION TITLE
Remove repeated words and create titles instead

### DISCUSSION THREAD

**Discussion by Sopor** [rxhanson/Rectangle#698] [Ideas]
I started to translate Rectangle and after some time i noticed that it repeats the same words in the menus.

For example, Thirds is at the end of every command instead of having a title like this:
![image](https://user-images.githubusercontent.com/5789283/147617313-14167e0a-2046-4b9c-90f2-2bceb93246e5.png)

Fourths i need to open, so why write Fourth and Fourths at the end of every line? I already know that i'm in the menu for Fourths. It will also make the menus smaller:
![image](https://user-images.githubusercontent.com/5789283/147617469-0d2f24dd-9755-431d-be0e-e04bdb112f63.png)

Same goes for the Sixths:
![image](https://user-images.githubusercontent.com/5789283/147617517-296fe953-4022-4251-b389-54ae6b75ab13.png)

I was going to do that for Fourths and Sixths when i translated Rectangle, but then i saw that the shortcuts settings looked weird, so i couldn't do anything about it. If Thirds, Fourths and Sixths get a title above every column it wouldn't be a problem anymore. Maybe something like this? Add a grey frame and add a title (i used a red frame to show it more clearly).
![image](https://user-images.githubusercontent.com/5789283/147618151-8a7d6f40-45e4-468d-939d-ec15b03633ee.png)

I haven't changed Thirds, but you get the point :)

Even `Next Display` and `Previous Display` could be using a title, so it become `Next`and `Previous` :)

Maybe this isn't possible due to other things, but i'm not a macOS user. I only wanted to translate Rectangle.

Yes, i can run macOS programs and most of the translations are done:
![image](https://user-images.githubusercontent.com/5789283/147619815-6cc43a8d-4d41-4a67-b81f-d9f9728f6958.png)

**Comment by rxhanson** [comment]
Thanks for looking to translate, and for posing the question.

Although this would clean up the UI quite a bit, there are a few reasons off the top of my head why I would prefer not to change these.

1. It is more "future proof" by keeping each menu item title unique. If I were to allow users to configure the menu entirely (a common request), it would require the unique titles.
2. I find that users actually do get tripped up on non-unique items. In my head I can already see the bugs filed saying that "first third" was actually triggering "first fourth". 
3. In English, "First Three" in the "Fourths" section is strange, especially compared to just "First Three Fourths" since it's a fraction. If I were to make an exception for just those, then we don't really gain that much in my opinion by still having the full title in some scenarios and not others.
